src/libs/Navigation/linkingConfig/createNormalizedConfigs.ts

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+/* eslint-disable @typescript-eslint/default-param-last */
+
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+
+/* eslint-disable no-param-reassign */
+
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/* eslint-disable @typescript-eslint/ban-types */
+// THOSE FUNCTIONS ARE COPIED FROM react-navigation/core IN ORDER TO AVOID PATCHING
+// THAT'S THE REASON WHY ESLINT IS DISABLED
```

Comment:
@Kicu @adamgrzybowski Is it suggested to use this code permanently? Why can we not use this from react-navigation? Is it not exposed or did we need to make some changes to this method? If we cannot get this merged upstream, can we find a different way so we avoid duplicating this?

Reply:
it's not exposed. We can ask somebody from react-navigation to expose it.

Reply:
Discussing more here https://expensify.slack.com/archives/C04878MDF34/p1721389874173849?thread_ts=1721350042.643069&cid=C04878MDF34